src/main/java/com/github/javabdd/JFactory.java

```diff
@@ -6277,110 +6292,92 @@ void varprofile_rec(int r, int[] varprofile) {
         varprofile_rec(HIGH(r), varprofile);
     }
 
-    double bdd_pathcount(int r) {
+    BigInteger bdd_pathcount(int r) {
         CHECK(r);
 
         miscid = CACHEID_PATHCOU;
 
         if (countcache == null) {
-            countcache = BddCacheD_init(cachesize);
+            countcache = BddCacheBI_init(cachesize);
         }
 
         return bdd_pathcount_rec(r);
     }
 
-    double bdd_pathcount_rec(int r) {
-        BddCacheDataD entry;
-        double size;
+    BigInteger bdd_pathcount_rec(int r) {
+        BddCacheDataBI entry;
 
         if (ISZERO(r)) {
-            return 0.0;
+            return BigInteger.ZERO;
         }
         if (ISONE(r)) {
-            return 1.0;
+            return BigInteger.ONE;
         }
 
-        entry = BddCache_lookupD(countcache, PATHCOUHASH(r));
+        entry = BddCache_lookupBI(countcache, PATHCOUHASH(r));
         if (entry.a == r && entry.c == miscid) {
-            return entry.dres;
+            return entry.bires;
         }
 
-        size = bdd_pathcount_rec(LOW(r)) + bdd_pathcount_rec(HIGH(r));
+        BigInteger size = bdd_pathcount_rec(LOW(r)).add(bdd_pathcount_rec(HIGH(r)));
 
         entry.a = r;
         entry.c = miscid;
-        entry.dres = size;
+        entry.bires = size;
 
         return size;
     }
 
-    double bdd_satcount(int r) {
+    BigInteger bdd_satcount(int r) {
         if (ZDD) {
             return bdd_pathcount(r);
         }
 
-        double size = 1;
-
         CHECK(r);
 
         if (countcache == null) {
-            countcache = BddCacheD_init(cachesize);
+            countcache = BddCacheBI_init(cachesize);
         }
 
         miscid = CACHEID_SATCOU;
-        if (!ZDD) {
-            size = Math.pow(2.0, LEVEL(r));
-        }
-
-        return size * satcount_rec(r);
-    }
-
-    double bdd_satcountset(int r, int varset) {
```

Comment:
Has this method been removed on purpose?

Reply:
Yes, it is not used anywhere, and is not public API.